index.js

```diff
@@ -1 +1,6 @@
 // Write your code in this file!
+const currentUser = 'Grace Hopper';
+const welcomeMessage = `Welcome to Flatbook, ${currentUser}!`;
+const excitedWelcomeMessage = welcomeMessage.toUpperCase();
+
+const shortGreeting = `Welcome, ${currentUser.slice(0,1)}!`;
```

Comment:
Any reason for the inconsistent line spacing?

Reply:
Not exactly. I just wanted some separation I guess. What are the usual conventions for line spacing in code?

Reply:
There usually aren't any strict conventions around spacing in this sense--mostly personal preference. I would say lean toward readability and consistency. 

I was mostly just curious if you added the space for a certain reason or not. Seems like you had one and usually that's all that matters, if you see what I mean.